### DISCUSSION TITLE
Code for Data Structure using C notes

### DISCUSSION THREAD

**Discussion by sakshamgarg6500** [sakshamgarg6500/Data-Structure-using-C-Notes#2] [General]
If you guys want I can upload the code that I have written during making these notes.

**Comment by noobshubham** [comment]
Hello, loved the notes you taken. Can you please tell me which app you used to take the note?

also can you please provide the quiz for this course. Thank You!

**Reply by sakshamgarg6500** [reply]
Hello Shubham
The application that I used for taking these notes is Goodnotes 5.
As you asked for quiz, my reply would be that just practice these algorithms randomly. Like take a pen and paper and pick any algorithm like insertion of node in circular linked list and try to write it from scratch. This method helped me a lot.

**Comment by noobshubham** [comment]
Okay, Thanks Buddy ✌️

**Comment by Piyush4567** [comment]
your ds notes are really good 
can u please reupload them 
as they are not appearing at moment

**Comment by debsouryadatta** [comment]
Thanks for these beautiful notes, so much helpful ❣️

**Reply by guru9607** [reply]
Can you provide a drive link as i am unable to view them

**Reply by debsouryadatta** [reply]
> Can you provide a drive link as i am unable to view them
> 

Just download the whole zipped code from the repo, all the notes will be available
